bcp47/tests/Data/BCP47/TrieSpec.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE OverloadedLists #-}
```

Comment:
I'm sure you considered this, but just noting that we tend to regret adding this and have it as an Avoid guide. I get that it saved a lot of churn below in all the `fromList` test cases, but (as my own import churn shows) you're not usually one to shy away from the "now" pain in favor of "later" gain :joy: 

**EDIT**: now that you have `fromNonEmpty` and changed `fromList`'s type, it seems like it could be about equal churn:

```diff
-      let trie = fromList [(en, "color"), (enGB, "colour")]
+      let Just trie = fromList [(en, "color"), (enGB, "colour")]
```

Reply:
That works. I'll switch to it. I tend to regret it in anything other than test code, but I do often regret it there too, so a mono-bind is a better choice.